app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  after_create :increment_post_likes_counter
+
```

Comment:
---

Hello,

I've been reviewing the `Like` model in your project, and I noticed that it's currently missing a method that's required as per the specifications. This method is supposed to update the likes counter for a post each time a new like is added or removed.

The requirement is as follows:

- Like model has a method that updates the likes counter for a post.

To address this requirement, you should define an instance method within the `Like` model. This method will be responsible for adjusting the `likes_counter` attribute in the associated `Post` model. Here’s an example of how you might write this method:

```ruby
class Like < ApplicationRecord
  belongs_to :post

  # Other code for the Like model

  # Method to update the post's likes counter
  def update_likes_counter
    post.increment!(:likes_counter)
  end
end
```

In the above example, the `increment!` method is used to add to the `likes_counter` directly and save the `Post` record with the new count. Make sure to invoke this method whenever a `Like` is created or destroyed to keep the count accurate.

Please implement this functionality and check that it's being triggered correctly within your application's flow. It's crucial for maintaining the integrity of your likes count, which is an important feature for user engagement on posts.

If you have any questions or face any issues while implementing this, feel free to reach out.

Looking forward to your update, and great job so far!

Best regards,

Temitope

---